server/odc-migrate/src/main/resources/init-config/init/regulation-rule-applying.yaml

```diff
@@ -1436,6 +1452,14 @@
     - 'OB_MYSQL'
     - 'MYSQL'
   propertiesJson: '{"${com.oceanbase.odc.builtin-resource.regulation.rule.sql-check.restrict-sql-affected-rows.allowed-max-sql-affected-count}":1000}'
+- enabled: 1
+  level: 1
+  rulesetName: ${com.oceanbase.odc.builtin-resource.regulation.ruleset.default-prod-ruleset.name}
+  ruleName: ${com.oceanbase.odc.builtin-resource.regulation.rule.sql-check.estimate-sql-affected-rows-failed.name}
+  appliedDialectTypes:
+    - 'OB_MYSQL'
+    - 'MYSQL'
+  propertiesJson: '{"${com.oceanbase.odc.builtin-resource.regulation.rule.sql-check.estimate-sql-affected-rows-failed.unable-to-estimate-sql-affected-count}":-1}'
```

Comment:
why should we add this property? I think there is no parameter for this rule

Reply:
right, actually this rule is no param, so `propertiesJson: {}` is what I should define ?